### ISSUE TITLE
Add_pop returns that the e-mail i just created is already created

### ISSUE THREAD

**Issue by juliocoliveira** [N1ghteyes/cpanel-UAPI-php-class#2]
When ever i use add_pop it does create the e-mail but also returns that the e-mail i just created exists already.

I made sure that the call is only run once by ajax and the function is only called once.

Returns: The account email@domain.com already exists!

**Comment by juliocoliveira** [comment]
I fixed that bug by changing curl_request.
From

``` PHP
$content = $this->curl_exec_follow($ch);
```

To:

``` PHP
   $maxredirect = 0;

    $content = $this->curl_exec_follow($ch, $maxredirect);
```

**Comment by N1ghteyes** [comment]
Hey, Sorry its taken me a while to get back, i've been on holiday :).

Pretty sure this has come up before, i'm glad you've found a fix for it. I will take a look at rolling max redirects into the code so others can do the same.

**Comment by N1ghteyes** [comment]
Closed as fixed in v1.1
